addons/io/org.openhab.io.homekit/README.md

```diff
@@ -102,6 +102,8 @@ HomeKit allows only a single pairing to be established with the bridge. This pai
 
 HomeKit requires a unique identifier for each accessory advertised by the bridge. This unique identifier is hashed from the Item's name. For that reason, it is important that the name of your Items exposed to HomeKit remain consistent.
 
+HomeKit listens on default port: 9124. IPv6 stack is preferred by default. If you have connection or detecting problems, try to prefer IPv4 stack. To prefer IPv4 stack, add this line to /etc/default/openhab2: ```_JAVA_OPTIONS="-Djava.net.preferIPv4Stack=true"```
```

Comment:
`/etc/default/openhab2` only exists if you use apt installation, but in no other case.
The correct suggestion would be to simply set the variable `JAVA_OPTS` (however this is done for the different OS's and installation options).

Reply:
@GeminiServer could you update this PR, note that I think that it could be really useful because I also spend some time once to fix some problems because it tried to run over ipv6